examples/lighting-app/lighting-common/gen/callback-stub.cpp

```diff
@@ -853,7 +853,7 @@ bool emberAfKeyEstablishmentCallback(EmberAfKeyEstablishmentNotifyMessage status
  * @param endpoint   Ver.: always
  * @param newValue   Ver.: always
  */
-void emberAfOnOffClusterLevelControlEffectCallback(uint8_t endpoint, bool newValue) {}
+//void emberAfOnOffClusterLevelControlEffectCallback(uint8_t endpoint, bool newValue) {}
 
```

Comment:
You can remove this code instead of commenting it. I'm going to remove it in #3803 anyway.

Reply:
Done.